testdata/project-v4-with-deploy-image/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Kubebuilder DevContainer",
+  "image": "golang:1.22-bullseye",
```

Comment:
Why need to be `golang:1.22-bullseye` and not `golang:1.22`

Reply:
No particular reason. Both point to Debian based image.

Reply:
So last use golang:1.22